eclipse-che/che-theia
# Rework reading pod's information for resource monitor plugin

Signed-off-by: svor <vsvydenk@redhat.com>

### What does this PR do?
Rework the process of reading pod's information for resource monitor plugin

### Screenshot/screencast of this PR
Tested on local minikube and on OCP 4.6.24

![screenshot-nimbus-capture-2021 04 22-13_19_27](https://user-images.githubusercontent.com/1271546/115703874-4dfe2580-a373-11eb-8214-9f832316a5ba.png)

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/18812


### How to test this PR?
This PR depends on https://github.com/eclipse-che/che-operator/pull/793 and https://github.com/eclipse/che/pull/19651

To test this PR need to deploy che with custom `che-operator` image which should include changes from https://github.com/eclipse-che/che-operator/pull/793 and custom `cheimage` which should include changes from https://github.com/eclipse/che/pull/19651. 

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### Happy Path Channel

HAPPY_PATH_CHANNEL=stable
